tools/serverpod_cli/lib/src/serverpod_cloud/local_cloud_server.dart

```diff
@@ -0,0 +1,117 @@
+import 'dart:async';
+import 'dart:io';
+
+import 'package:serverpod_cli/src/logger/logger.dart';
+
+abstract final class LocalCloudServer {
```

Comment:
Suggestion: Change the name of this class, it is a little confusing right now. Something like `TokenListenerServer` or similar describes the functionality better.